src/translator/definitions.h

```diff
@@ -34,14 +34,14 @@ struct MemoryBundle {
   MemoryBundle(MemoryBundle &&from){
     model = std::move(from.model);
     shortlist = std::move(from.shortlist);
-    vocabs = std::move(vocabs);
+    vocabs = std::move(from.vocabs);
     ssplitPrefixFile = std::move(from.ssplitPrefixFile);
   }
 
   MemoryBundle &operator=(MemoryBundle &&from) {
     model = std::move(from.model);
     shortlist = std::move(from.shortlist);
-    vocabs = std::move(vocabs);
+    vocabs = std::move(from.vocabs);
     ssplitPrefixFile = std::move(from.ssplitPrefixFile);
     return *this;
   }
```

Comment:
Can you check if the compiler assigned move and copy-constructors/assignments are good enough? All members are `AlignedMemory` which are only move-ctor/assign-able. Trivially generated ones should be okay?

This is more room for error.